config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -5514,3 +5514,18 @@ rubrics.integration.token-secret=12345678900909091234
 # END FILE CONVERSION SERVICE
 # ###############################
 
+# ###############################
+# Ignite
+# ###############################
+
+# Sets Ignite metrics update frequency in milliseconds.
+# Default: 2000L
+# ignite.metrics.update.freq=5000
+
+# Sets frequency of metrics log print out. If 0, metrics print out is disabled.
+# Default: 60000L
+# ignite.metrics.log.freq=600000
```

Comment:
Can we make the default `0`? I don't see the point of having this printed out every 60 seconds OOTB.

Reply:
i agree with bjones86. @ern?

Reply:
yep was going to disable this later

Reply:
I'm fine either way making it 0 now, I can update those defaults. If we still want to see the logging on some of the nighties we could adjust this. 

I think even going to something like 5 or 10 minutes (instead of 1 minute) would be easier to parse.